server/Cargo.lock

```diff
@@ -2956,9 +2968,9 @@ dependencies = [
 
 [[package]]
 name = "redis"
-version = "0.23.0"
+version = "0.23.2"
```

Comment:
Don't update redis, this was part of the issue we had today.

Why are you updating things anyway? Just do the cargo deny to block the serde version.